deploy/docker/Dockerfile

```diff
@@ -46,3 +46,8 @@ RUN mkdir -p /app/uploads/ && \
     chmod o=rX -R /app && \
     chmod 700 /app/uploads/ && \
     chown nobody:nobody /app/uploads/
+
+ENV PYTHONPATH=/app
+WORKDIR /app
+
+CMD ["/bin/sh", "/app/start.sh"]
```

Comment:
`/bin/sh` seems to be unnecessary here :thinking:

Reply:
Applied (and set shebang to `#!/bin/sh`, we don't have bash in that alpine-based image)